ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -0,0 +1,95 @@
+# Exportable code - demo package
+
+Demo package contains simple demo to get and visualize result of model inference.
+Name of package is a name of model which was deployed.
+
+## Structure of generated package:
+
+* [model](../model)
```

Comment:
Do these links have to be links or plain text would be enough? Who and how will render this md file?